tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -13,7 +13,7 @@
 # instantiate Cumulus logger
 LOGGER = CumulusLogger(name="ORCA")
 
-CONFIG_EXCLUDE_FILE_TYPES_KEY = "excludeFileTypes"
+CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS = "orcaExcludedFileExtensions"
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention

Reply:
updated